.codenvy/project.json

```diff
@@ -1 +1 @@
-{"mixinTypes":["contribution"],"builders":{"configs":{},"default":"maven"},"runners":{"configs":{"system:/java/web/jetty9":{"ram":1000,"variables":{},"options":{}}},"default":"system:/java/web/jetty9"},"type":"maven","attributes":{"contribute_branch":["master"],"languageVersion":["1.6"],"contribute_mode":["contribute"],"language":["java"]}}
\ No newline at end of file
+{"mixinTypes":["contribution"],"builders":{"configs":{},"default":"maven"},"runners":{"configs":{"system:/java/web/jetty9":{"ram":1000,"variables":{},"options":{}}},"default":"system:/java/web/jetty9"},"description":"The frame for uw applications","type":"maven","attributes":{"contribute_branch":["upgrade_to_uw_ui_toolkit_v0.3.1"],"languageVersion":["1.6"],"contribute_mode":["contribute"],"language":["java"]}}
```

Comment:
disregard this change, might decide we don't want these project.json files in our projects.